src/utils/setup.middleware.js

```diff
@@ -55,7 +55,9 @@ exports.initLocals = function( req, res, next ) {
 			{ title: 'How you can help', href: '/page/how-you-can-help' },
 			{ title: 'How businesses and organizations can help', href: '/page/how-businesses-and-organizations-can-help' },
 			{ title: 'Experienced families', href: '/page/experienced-families' },
-			{ title: 'Weekend Family Connections', href: '/page/weekend-family-connections' }
+			{ title: 'Weekend Family Connections', href: '/page/weekend-family-connections' },
+			{ title: 'The Neighborhood', href: '/page/the-neighborhood' },
+			{ title: 'Shop', href: 'store.mareinc.org' }
```

Comment:
This href value is being treated as a relative path, when testing locally this link navigates me to `http://localhost:3000/store.mareinc.org`.

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/5726612/153673330-d7807a2f-92cb-4161-897f-8334e0fb84ea.png">

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/5726612/153673411-a79192c4-769c-41eb-bcbd-91d8e1f2e607.png">


In production, the link would behave similarly and navigate the user to `https://www.mareinc.org/store.mareinc.org`

Using an absolute URL should solve the issue, e.g. `https://store.mareinc.org`